playbooks/archivematica-centos7/singlenode.yml

```diff
@@ -7,12 +7,41 @@
       tags:
         - "always"
 
-    - name: "Set SELinux to Permissive"
-      command: "setenforce Permissive"      
-      become: "yes"
+   - name: "Selinux boolean 1"
+     become: "yes"
+     seboolean:
+       name: "httpd_can_network_connect"
+       state: "yes"
+       persistent: "yes"
+     when: ansible_selinux.status == "enabled"
 
-    - name: "Enable epel repository"
-      yum:
+   - name: "Selinux boolean 2"
+     become: "yes"
+     seboolean:
+       name: "httpd_can_network_connect_db"
+       state: "yes"
+       persistent: "yes"
+     when: ansible_selinux.status == "enabled"
+
+   - name: "Selinux boolean 3"
+     become: "yes"
+     seboolean:
+       name: "httpd_setrlimit"
+       state: "yes"
+       persistent: "yes"
+     when: ansible_selinux.status == "enabled"
+
+   - name: "Selinux seport"
+     become: "yes"
+     seport:
+       ports: 8000,8001
```

Comment:
Do you want to quote this so it's consistent with our style of writing YAML files? It's definitely a string.